hpi-swt2/lecture-portal
# Feature/#71 question upvotes

# Resolves #71 

This PR introduces the feature of upvoting questions.

Acceptance Criteria:
- [x] I can not upvote my own questions
- [x] I can upvote a question only a single time
- [x] Upvotes become visible to other users (see mockup)
- [x] Upvoting adheres to soft real-time constraints


Testing steps:
- as a student, go to "Questions" (/questions) and upvote questions from other students
- as a lecturer, go to "Questions" (/questions) and see upvotes
- rspec

Steps needed for migration:
- bundle install
- rails db:migrate (if that doesn't work, try rails db:reset)

---

Definition Of Done:
